src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddleware.cs

```diff
@@ -192,6 +194,22 @@ private Task DisplayException(HttpContext context, Exception ex)
 
         private Task DisplayRuntimeException(HttpContext context, Exception ex)
         {
+            var endpoint = context.Features.Get<IEndpointFeature>()?.Endpoint;
+
+            EndpointModel endpointModel = null;
+            if (endpoint != null)
+            {
+                endpointModel = new EndpointModel();
+                endpointModel.DisplayName = endpoint.DisplayName;
+                endpointModel.RouteName = endpoint.Metadata.GetMetadata<IRouteNameMetadata>()?.RouteName;
```

Comment:
I wonder if we really need route name here. It's only used by link generation, and it doesn't **really** help you identify what code failed.

Reply:
Ok, I'll get rid of it.

Is there any metadata that you want to add? HttpMethod might be useful